### ISSUE TITLE
Material Table Site Hacked?

### ISSUE THREAD

**Issue by bgbbmm** [material-table-core/website#27]
I went to the website linked for this repo: [https://material-table-core.com/](https://material-table-core.com/)

When I go there, I am presented with a Core Flower Decor website about ordering flowers. Is the domain hacked? Was it never setup? Someone should probably look into that.

**Comment by Domino987** [comment]
No, this was the side from the previous owner. The current one is https://material-table-core.github.io/
